src/Swagger/ApiDocumentationBuilder.php

```diff
@@ -79,13 +79,14 @@ public function getApiDocumentation() : array
     {
         $classes = [];
         $operation = [];
-        $operation['item'] = [];
-        $operation['collection'] = [];
 
         $itemOperationsDocs = [];
         $definitions = [];
+        $basePath = $this->urlGenerator->generate('api_hydra_entrypoint');
```

Comment:
Can't you pass the `ApiPlatform\Core\Api\UrlGeneratorInterface::REL_PATH);` as second parameter and remove the addition at line 167 instead?